### DISCUSSION TITLE
detect the position of the object in 3D

### DISCUSSION THREAD

**Discussion by StefanoGarzoni** [roboflow/notebooks#199] [Q&A]
up to now I have tried to use the classic yolo method to try to detect objects in real time via the webcam. After many problems I finally did it, but now I wonder if it is possible to do it in 3D. Let me explain better, I would like to be able to identify the position in space for example of my basketball being thrown by seeing the x y z coordinates in real time.... Does anyone know if it can be done, if so how to do it and if perhaps there are link or video where this topic is covered in depth? My growth in recent months for what is now my passion I owe above all to you so in any case, thank you very much in advance!
Please help me.

**Comment by SkalskiP** [comment]
Is that more or less what you are looking for: https://github.com/SkalskiP/sport#--3d-football-players-pose-estimation-with-yolov7 ?

**Reply by StefanoGarzoni** [reply]
not perfectly that but ok, really charming! it's incredible how I like any project idea... anyway I was looking for something so that maybe it could simply give me the position so yes with the 3 spatial coordinates of any object (one of those recognized by yolo base for example) I don't know if you pass through here anyway, but in any case, how do you do this thing (the boy who dribbles)? is there a tutorial, code, explanation or anything?

**Reply by SkalskiP** [reply]
@Stefano0513, do you mean the visualization?

**Reply by StefanoGarzoni** [reply]
> @Stefano0513, do you mean the visualization? 

visualization of data x y z if this is that you mean
